core/Symbols.h

```diff
@@ -451,6 +451,11 @@ class Symbol final {
         flags |= Symbol::Flags::CLASS_OR_MODULE_LINEARIZATION_COMPUTED;
     }
 
+    inline void unsetClassLinearizationComputed() {
```

Comment:
could this method be private(and only used by that helper method suggested in https://github.com/sorbet/sorbet/pull/1815/files#r326829534)